examples/Dropdown.test.tsx

```diff
@@ -11,19 +12,29 @@ const today = new Date(2015, 6, 1);
 beforeAll(() => jest.setSystemTime(today));
 afterAll(() => jest.useRealTimers());
 
-beforeEach(() => {
-  render(<Dropdown />);
-});
-
 test("should display the month dropdown", () => {
+  render(<Dropdown />);
   expect(monthDropdown()).toBeInTheDocument();
 });
 
 test("should display the year dropdown", () => {
+  render(<Dropdown />);
   expect(yearDropdown()).toBeInTheDocument();
 });
 
+test("should display the month and year dropdowns in the correct order", () => {
+  const { rerender } = render(<Dropdown />);
+  const comparePosition = () =>
+    monthDropdown().compareDocumentPosition(yearDropdown());
+
+  expect(comparePosition()).toBe(Node.DOCUMENT_POSITION_FOLLOWING);
+
+  rerender(<Dropdown locale={ja} />);
+  expect(comparePosition()).toBe(Node.DOCUMENT_POSITION_PRECEDING);
```

Comment:
This new feature will also change the way date text is displayed, so we can assert that the text is rendered in the right order using `toHaveAccessibleName`, like the other tests.
But to do so, we also need to update the accessible name of the calendar, could you update the [`labelGrid`](https://github.com/gpbl/react-day-picker/blob/main/src/labels/labelGrid.ts) to have the same dynamic logic based on `locale`?
Please extract the logic to compute the month/year order based on `locale` to a separate function.